modules/administration-guide/partials/proc_restoring-a-che-instance-by-chectl-using-a-checluserbackup-custom-object.adoc

```diff
@@ -0,0 +1,35 @@
+[id="restoring-a-{prod-id-short}-instance-by-{prod-cli}-using-a-checluserbackup-custom-object_{context}"]
+= Restoring a {prod-short} instance by {prod-cli} using a CheClusterBackup custom object
+
+//To restore a state described by an existing backup Custom Resource:
+
+If a `CheClusterBackup` custom object is used to create a backup snapshot of a {prod-short} instance, you can restore the {prod-short} instance by entering the name of the `CheClusterBackup` custom object as an argument with the `{prod-cli} server:restore` command:
```

Comment:
Such an object is created anyway (even if use chectl). So, if it wasn't deleted, user may find it.